Compile error on Ubuntu 20.04

```
[100%] Building CXX object crazyflie_ros/sim_cf/crazyflie_gazebo/CMakeFiles/rotors_gazebo_imu_plugin.dir/src/gazebo_imu_plugin.cpp.o
c++: fatal error: Killed signal terminated program cc1plus
compilation terminated.
make[2]: *** [crazyflie_ros/sim_cf/crazyflie_gazebo/CMakeFiles/gazebo_cfHandler_plugin.dir/build.make:63: crazyflie_ros/sim_cf/crazyflie_gazebo/CMakeFiles/gazebo_cfHandler_plugin.dir/src/gazebo_cfHandler_plugin.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:7105: crazyflie_ros/sim_cf/crazyflie_gazebo/CMakeFiles/gazebo_cfHandler_plugin.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
/home/bitcraze/catkin_ws/src/crazyflie_ros/sim_cf/crazyflie_gazebo/src/gazebo_odometry_plugin.cpp: In member function 'virtual void gazebo::GazeboOdometryPlugin::Load(gazebo::physics::ModelPtr, sdf::ElementPtr)':
/home/bitcraze/catkin_ws/src/crazyflie_ros/sim_cf/crazyflie_gazebo/src/gazebo_odometry_plugin.cpp:100:48: error: 'CV_LOAD_IMAGE_GRAYSCALE' was not declared in this scope
  100 |     covariance_image_ = cv::imread(image_name, CV_LOAD_IMAGE_GRAYSCALE);
      |                                                ^~~~~~~~~~~~~~~~~~~~~~~
```

--- a/crazyflie_gazebo/src/gazebo_odometry_plugin.cpp
+++ b/crazyflie_gazebo/src/gazebo_odometry_plugin.cpp
@@ -97,7 +97,7 @@ GazeboOdometryPlugin::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf)
   if (_sdf->HasElement("covarianceImage")) {
     std::string image_name =
       _sdf->GetElement("covarianceImage")->Get<std::string>();
-    covariance_image_ = cv::imread(image_name, CV_LOAD_IMAGE_GRAYSCALE);
+    covariance_image_ = cv::imread(image_name, cv::IMREAD_GRAYSCALE);
     if (covariance_image_.data == NULL)
       gzerr << "loading covariance image " << image_name << " failed"
             << std::endl;

